src/window.rs

```diff
@@ -874,3 +896,15 @@ pub enum Theme {
     Light,
     Dark,
 }
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum RequestUserAttentionType {
+    Critical,
+    Informational,
+}
+
+impl Default for RequestUserAttentionType {
+    fn default() -> Self {
+        RequestUserAttentionType::Critical
```

Comment:
What's the point of using `Critical` by default, I think `Informational` should be default, since it's more relaxed

Reply:
Good point changed it!